autoscale/src/main/java/com/sequenceiq/periscope/service/PeriscopeMetricService.java

```diff
@@ -32,4 +39,26 @@ protected void init() {
     protected Optional<String> getMetricPrefix() {
         return Optional.of(METRIC_PREFIX);
     }
+
+    @Override
+    protected boolean gaugeMetric(Metric metric) {
+        return metric.getMetricName().contains("state") || metric.getMetricName().contains("leader") || metric.getMetricName().contains("threadpool")
+                || metric.getMetricName().contains("scaling.activity");
```

Comment:
Can we define a constant will all these values and update this code. In future, only value for constant will change.

Reply:
Updated